bin/writeReadme.js

```diff
@@ -0,0 +1,70 @@
+const fs = require("fs");
+const path = require("path");
+const spawn = require("child_process").spawn;
+
+function getGitSha() {
+  return new Promise((resolve, reject) => {
+    const proc = spawn("git", ["log", "--format=%H", "-n", "1"]);
+
+    let version = "";
+
+    proc.stdout.on("data", data => {
+      version += data;
+    });
+    proc.on("close", code => resolve(version.trim()));
+    proc.on("error", error => {
+      if (error.code == "ENOENT") {
+        reject(new Error("Could not find git."));
+        return;
+      }
+      reject(error);
+    });
+  });
+}
+
+function getPackageVersion(name) {
+  try {
+    const json = require(`${name}/package.json`);
+    const { version } = json;
+    return { name, version };
+  } catch (ex) {
+    return { name, version: "n/a" };
+  }
+}
+
+const packageOfInterest = [
+  "babel-plugin-transform-es2015-modules-commonjs",
+  "babel-preset-react",
+  "react",
+  "react-dom",
```

Comment:
we exclude some of these from the bundle: 

https://github.com/ahuth/debugger.html/blob/fix-warnings/webpack.config.js#L53-L61

Reply:
right, we might need to know difference anyway

Reply:
list is incomplete too